### DISCUSSION TITLE
Reversed compiler

### DISCUSSION THREAD

**Discussion by KarelJanVanHaute** [tailwindlabs/tailwindcss#10010] [Help]
Is there an option to reverse compile? So from css to tailwind class. 
Say you have the css property `display: flex`. Is there an option in tailwind to convert this to `flex` as a simple example.

I'm playing with the idea of writing a vscode plugin to easily convert classes on a html tag to tailwind classes.
So you select a .css file for the current styling. Then you select a class name in the editor. And with a shortcut, this gets converted to tailwind classes following the css properties in linked .css file.

**Comment by adamwathan** [comment]
Hey! I think this tool by community member @mpociot is pretty close to what you’re looking for:

https://beyondco.de/software/windy

Really well done project 👍🏻

**Reply by KarelJanVanHaute** [reply]
Yes indeed I found this one already. It's close, but then you still have to view it in the browser first. It would be nice if you could do it in your editor. But tailwind doesn't have a feature of a reverse lookup?
